src/main/resources/assets/twilightforest/patchouli_books/guide/en_us/entries/treasures/foods.json

```diff
@@ -9,25 +9,33 @@
       "type": "text",
       "text": "Food glorious food"
     }, {
-      "type": "text",
+      "type": "spotlight",
+      "title": "Hydra Chop",
+      "item": "twilightforest:hydra_chop",
       "advancement": "twilightforest:alt/treasures/foods/hydra_chop",
-      "text": "things and stuff"
+      "text": "A filling meal with just a single Hydra Chop. Eating this potent meat fills your hunger greatly and helps in health regeneration. Fitting for those that have conquered such a mighty beast."
```

Comment:
`Eating this potent meat fills your hunger greatly and helps in health regeneration.`

Perhaps something more like `Eating this dense meat satiates hunger so potently that it also provides temporary rapid healing`, since it's more of an in-world perspective. Also IIRC we should be avoiding self-referential wording too

Reply:
Suggest: "Eating this hearty chop will satisfy almost any hunger and is so potent it will provide temporary, rapid healing"

Reply:
`almost any hunger` is strange wording, and it will only restore 9 shanks on the bar too.